frontend/express/public/core/user-management/stylesheets/_drawer.scss

```diff
@@ -0,0 +1,266 @@
+.user-management-drawer-content {
+
+  &__top-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
+
+  &__bottom-section {
+    display: flex;
+    margin-top: 28px;
+    flex-direction: column;
+    justify-content: space-between;
+  }
+
+  &__inputs-area {
+    //width: calc(100% - 310px);
+
+    &__input-wrapper {
+      margin-top: 10px;
+    }
+  }
+
+  &__label {
+    display: flex;
+    justify-content: space-between;
+
+    &--pl-10 {
+      padding-left: 10px;
+    }
+
+    &--text-secondary {
+      color: #81868D;
+    }
+  }
+
+  &__label-text {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 13px;
```

Comment:
We've already spoken about custom styling (e.g. font size, weight), so it is ok for now. In the future, however, these should be deleted and elements should be styled with predefined global classes (.text-medium, .text-small, ...).

Reply:
yes, this change in my bucket list but as we talked it's not a good time for this. I'll use global modifiers on the next usages.